models/tmp/stg_shopify__refund_tmp.sql

```diff
@@ -1,6 +1,15 @@
 --To disable this model, set the shopify__using_refund variable within your dbt_project.yml file to False.
```

Comment:
Do you mind explaining why this model is different than the rest?

Reply:
if i remember correctly, a request to do this came from a customer to @fivetran-joemarkiewicz ... joe would you mind elaborating here if i didn't make that up lol

Reply:
The refund tables are only synced once a refund is provided. Therefore, a customer could go months without refunding anything and never use that table. However, the second a refund is provided this table will be synced.

The experience from the customer was that it was difficult to maintain when refunds started. This is not a "I don't use this feature" as it is always used, but will only start syncing once a refund is provided. Customers shouldn't have to worry about managing their dbt project once refunds are processed. Therefore we took the approach of having this refund table be one that syncs as empty if it is not populated, then seamlessly switches to the source table once it exists.